### PR TITLE
move terraform how-to into the official docs site

### PR DESCRIPTION
**Pull request by danbudris**
*Issue #, if available:*

*Description of changes:*
Move the guide to managing a cluster with terraform from `examples` to the `tasks/clusters` section of the docs.

Add a section to the cluster mgmt overview which notes the ability to manage a cluster with terraform, including a link to the docs.

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/tasks/cluster/cluster-terraform.md [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+title: "Manage cluster with Terraform"
+linkTitle: "Manage cluster with Terraform"
+weight: 30
+date: 2017-01-05
+description: >
+  Use Terraform to manage EKS Anywhere Clusters
+---
+
+## Using Terraform to manage an EKS Anywhere Cluster (Optional)
+This is guide explains how you can use Terraform to manage and modify an EKS Anywhere cluster.
```
Comment (chrisnegus) [inline docs/content/en/docs/tasks/cluster/cluster-terraform.md:11]:
```suggestion
This guide explains how you can use Terraform to manage and modify an EKS Anywhere cluster. 
```

File: docs/content/en/docs/tasks/cluster/cluster-terraform.md [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
+date: 2017-01-05
+description: >
+  Use Terraform to manage EKS Anywhere Clusters
+---
+
+## Using Terraform to manage an EKS Anywhere Cluster (Optional)
+This is guide explains how you can use Terraform to manage and modify an EKS Anywhere cluster. 
+The guide is meant for illustrative purposes and is not a definitive approach to building production systems with Terraform and EKS Anywhere.
+
+At its heart, EKS Anywhere is a set of Kubernetes CRDs which define an EKS Anywhere cluster,
```
Comment (chrisnegus) [inline docs/content/en/docs/tasks/cluster/cluster-terraform.md:14]:
```suggestion
At its heart, EKS Anywhere is a set of Kubernetes CRDs, which define an EKS Anywhere cluster,
```

File: docs/content/en/docs/tasks/cluster/cluster-terraform.md [hunk 3]
```diff
@@ -1,13 +1,23 @@
...
+and a controller which moves the cluster state to match these definitions.
```
Comment (chrisnegus) [inline docs/content/en/docs/tasks/cluster/cluster-terraform.md:15]:
```suggestion
and a controller, which moves the cluster state to match these definitions.
```

**Review by chrisnegus — COMMENTED** [review]
Just a couple of small editing suggestions. Otherwise, /lgtm.

**Comment by jaxesn** [comment]
can we backport to 0.8?

**Pull request merged**